tools/bench/functional/permutor.py

```
@@ -0,0 +1,103 @@
+
+def clean_dir(dir):
+    if dir.exists():
+        shutil.rmtree(str(dir))
+
+
+def make_tarfile(outdir, outfile=None):
+    if MAKE_TAR_FILE and cpp.mpi_rank() == 0:
+        path = outdir.parent
+        outfile = outfile or outdir.name
+        args = ["tar", "czf", f"{path/outfile}.tar.gz", "-C", str(path), outdir.name]
+        ec = subprocess.call(args)
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)